daemon/src/tests/gossip.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
This file is currently located at `daemon/src/tests/gossip.rs`. I believe it should be located at `daemon/tests/gossip.rs`. Otherwise these tests are never run.

Reply:
Ahahaha thanks for the catch :smile: